core/src/main/scala/com/rallyhealth/vapors/factfilter/evaluator/InterpretExprAsFunction.scala

```diff
@@ -178,6 +190,7 @@ final class InterpretExprAsFunction[F[_] : Foldable, V, P]
       mapFn(mapInput)
     }
     val allValues = allResults.map(_.output.value)
+    // TODO: Shouldn't I only look at the evidence of results that aren't empty?
```

Comment:
Wait... this is the `.map` operator... technically, every input should have exactly 1 output, so there is no higher-kinded definition of an "empty" output. I should probably remove this TODO.